routes/screeningmovies.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+import { fetchScreenings } from '../utils/screeningUtils.js';
+import { mapScreenings } from '../utils/screeningUtils.js';
+import { frontpageScreening } from '../utils/screeningUtils.js';
+// import { fetchScreenings } from '../utils/screeningUtils.js';
+const screeningRouter = express.Router();
+
+
+
+
+
+
+
+screeningRouter.get('/screenings', async (req, res) => {
+    try {
+        const Screenings = await fetchScreenings()
```

Comment:
By convention, in Javascript we normally do not use capitalized words as variable names, so I would call this variable `screenings` instead of `Screenings`.

Note how the GitHub syntax highlighter event colors this variable yellow, because it assumes it's something special (e.g. a class) instead of a variable.